### ISSUE TITLE
Non-header nif

### ISSUE THREAD

**Issue by rioaje** [niftools/nifxml#107]
i stumbled a new versionless nif files from game Twin Saga / Astral Realm. in this case it doesn't seems to be the same as https://github.com/niftools/nifskope/issues/187

since the aura kingdom versionless 20.3.1.2 header were

> Gamebryo File Format\x0A\x02\x01\x03\x14\x01

while this one is 

> Gamebryo File Format\x0A\x01\x02\x03\x14\x01

> Gamebryo File Format\x0A\x02\x02\x03\x14\x01

[NIF.zip](https://github.com/niftools/nifxml/files/13325038/NIF.zip)

**Comment by Candoran2** [comment]
That look to be version 20.3.2.1 (`\x01\x02\x03\x14`) and version 20.3.2.2 (`\x02\x02\x03\x14`). Do you know which of those is the most common for each of those games?

Aside from just adding them, it seems like they will also need to be used in a couple places where the 20.3.1.2 header version is used as condition (such as block hash array vs block type array in the header).

**Comment by rioaje** [comment]
> Do you know which of those is the most common for each of those games?

im not really sure with this question, but are you asking how many of version 20.3.2.1 and 20.3.2.2 in the game data? if yes then. 

in folder char there is 2171 NIF file in total. there is 243 version 20.3.2.1 and 343 version 20.3.2.2

**Comment by Candoran2** [comment]
> im not really sure with this question, but are you asking how many of version 20.3.2.1 and 20.3.2.2 in the game data? if yes then.

Kind of. See, for each game, we need to know which nif version should be considered the default. If all of them are one specific version, then that's easy. But if a game contains multiple, then one metric to decide that by is by just taking the most common nif version.

**Comment by Nonstop1225** [comment]
Yeah I don't know much about them, just that they seem a bit different from Aura Kingdom. Because with 20.3.1.2 you could still see stuff like Scene Root text and what not if you opened it, these you can't see anything. But well I would say 20.3.2.2 is the main one used of the new version. There are over 6000 of them in biology animations.
